### PR TITLE
Add missing MAX macros, ensure that i3 can be compiled without pango

### PR DESCRIPTION
**Pull request by Xarthisius**
This PR ensures that i3 works without pango

### CODE REVIEW

File: libi3/font.c [hunk 1]
```diff
@@ -397,9 +397,11 @@ static void draw_text_xcb(const xcb_char2b_t *text, size_t text_len, xcb_drawabl
 void draw_text(i3String *text, xcb_drawable_t drawable, xcb_gcontext_t gc,
                xcb_visualtype_t *visual, int x, int y, int max_width) {
     assert(savedFont != NULL);
+#if PANGO_SUPPORT
```
Comment (Airblader) [inline libi3/font.c]: Why is this necessary?

Comment (Xarthisius) [inline libi3/font.c]: Cause if fails with `undefined root_visual_type` without it

Comment (Airblader) [inline libi3/font.c]: Right, we only need this for pango. Thanks.

**Comment by stapelberg** [comment]
Can you rebase please? This PR can’t be merged currently.

**Comment by Xarthisius** [comment]
@stapelberg It's rebased (3rd time) any chance of merging?
